tools/ansible/tasks/logging/test/logging_test.go

```diff
@@ -1,18 +1,26 @@
 package test
 
 import (
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 	"os/exec"
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
+// Pining down version of Ruby Devkit to 2.5, 64 bit version
+// Ruby installation path kept default C:\
+var rubyExecutablePath = "C:\\Ruby25-x64\\bin\\ruby.exe"
+var rubyVersion = "2.5.7"
+
 // Making this a map so that if we want a specific version, the value can be the version
 var requiredPluginList = map[string]string{
-	"elasticsearch-transport":                  "",
-	"elasticsearch-api":                        "",
-	"elasticsearch":                            "",
+	// Pining down version of Fluentd to 1.5
+	"fluentd":                 "1.5",
```

Comment:
Why is fluentd plugin versioning in a different format than other plugins?

Reply:
We are only checking the version for Fluentd plugin because the playbook has an option to specify the version of Fluentd installed. We are not checking the versions for the other plugins